tests/unit/nupic/algorithms/sdr_classifier_test.py

```diff
@@ -24,13 +24,21 @@
 
 import cPickle as pickle
 import random
+import tempfile
 import types
 import unittest2 as unittest
 
 import numpy
 
 from nupic.algorithms.sdr_classifier import SDRClassifier
 
+try:
+  import capnp
+except ImportError:
+  capnp = None
```

Comment:
Repeated pattern as before.  Another reason this should be implemented in a higher level api.